x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.ts

```diff
@@ -194,15 +210,22 @@ export const buildQueryExceptions = ({
   query,
   language,
   lists,
+  exclude = true,
 }: {
   query: Query;
   language: Language;
-  lists: ExceptionListItemSchema[] | undefined;
+  lists: Array<ExceptionListItemSchema | CreateExceptionListItemSchema> | undefined;
+  exclude?: boolean;
```

Comment:
This is needed because the logic is negated when adding an exception, but when closing alerts we don't want to do that.